modules/selenium/src/test/java/org/testcontainers/junit/BrowserWebDriverContainerTest.java

```diff
@@ -82,6 +83,25 @@ public void createContainerWithoutShmVolume() {
         }
     }
 
+    /**
+     * Test that getWebDriver has a sufficient default timeout on the latest images.
+     * If this test fails occasionally, this may indicate that the default timeout specified in getWebDriver is too low.
+     * This may occur on new versions of selenium which have been shown to be slower
+     */
+    @Test
+    public void getWebDriverShouldNotTimeout() {
+        try (
+            BrowserWebDriverContainer webDriverContainer = new BrowserWebDriverContainer<>(
+                DockerImageName.parse("selenium/standalone-firefox:latest")
+            )
+                .withCapabilities(new FirefoxOptions())
+        ) {
+            webDriverContainer.start();
+            // Should not throw a timeout exception
+            webDriverContainer.getWebDriver();
+        }
+    }
+
```

Comment:
This test could notify us if our timeout is too conservative. I am, however, unsure about using the `latest` tag as it may make our builds more unpredictable and possibly more flaky.

If you think we don't need it I can remove.